compute/functions/how-to/create-a-functions-namespace.mdx

```diff
@@ -27,7 +27,7 @@ This page shows you how to create your first functions namespace. [Namespaces](/
 
 3. Complete the following steps in the wizard: 
 
-    - Enter a **name** and optionally a **description** for your namespace. The name must only contain alphanumeric characters.
+    - Enter a **name** and optionally a **description** for your namespace. The first character of your namespace must be a letter. It can only contain letters, numbers and dashes. No other special characters are permitted. The namespace is case sensitive and cannot be edited after it was created.
```

Comment:
```suggestion
    - Enter a **name** and optionally a **description** for your namespace. The first character of your namespace must be a letter. It can only contain letters, numbers, and dashes. No other special characters are permitted. The namespace is case-sensitive and cannot be edited after its creation.
```